src/components/dapp-staking/dapp/ProjectDetails.vue

```diff
@@ -113,52 +125,55 @@
                 <span class="text--tooltip">{{ $t('common.twitter') }}</span>
               </q-tooltip>
             </button>
+            <button v-if="it.type === CommunityType.Reddit" class="box--share btn--primary">
+              <div class="icon--social">
+                <a :href="it.handle" target="_blank" rel="noopener noreferrer">
+                  <astar-icon-base :icon-name="CommunityType.Reddit">
+                    <astar-icon-reddit />
+                  </astar-icon-base>
+                </a>
+              </div>
+              <q-tooltip>
+                <span class="text--tooltip">{{ $t('common.reddit') }}</span>
+              </q-tooltip>
+            </button>
+            <button class="box--share btn--primary">
+              <div class="icon--social">
+                <a :href="it.handle" target="_blank" rel="noopener noreferrer">
+                  <astar-icon-base viewBox="0 0 18 18" :icon-name="CommunityType.Facebook">
+                    <astar-icon-facebook />
+                  </astar-icon-base>
+                </a>
+              </div>
+              <q-tooltip>
+                <span class="text--tooltip">{{ $t('common.facebook') }}</span>
+              </q-tooltip>
+            </button>
+            <button v-if="it.type === CommunityType.YouTube" class="box--share btn--primary">
+              <div class="icon--social">
+                <a :href="it.handle" target="_blank" rel="noopener noreferrer">
+                  <astar-icon-base :icon-name="CommunityType.YouTube">
+                    <astar-icon-youtube />
```

Comment:
Let's fix the hover styling for YouTube icon
![image](https://user-images.githubusercontent.com/92044428/201338351-d9c46547-0d52-40b2-af95-cd1ffea68c57.png)

Reply:
<img width="150" alt="스크린샷 2022-11-13 오후 9 55 17" src="https://user-images.githubusercontent.com/1606820/201522736-498dbe4d-8ceb-46aa-99c7-c3852659d2eb.png">

Fixed like this